src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1165,6 +1165,12 @@ private bool isValidStringRef(CORINFO_MODULE_STRUCT_* module, uint metaTOK)
         private bool shouldEnforceCallvirtRestriction(CORINFO_MODULE_STRUCT_* scope)
         { throw new NotImplementedException("shouldEnforceCallvirtRestriction"); }
 
+        private string getStringLiteral(CORINFO_MODULE_STRUCT_* module, uint metaTOK, ref int length)
```

Comment:
Does returning a `string` here mean that there's an expectation that RyuJIT will CoTaskMemFree the memory? (I.e once we implement this, there will be a memory leak.)

Maybe we should just treat this the same as the JitInterface methods that return `const char*` (i.e. have it return an actual pointer that we can clean up when we're done compiling).

Reply:
ah, ok, changed to `const char*` in the `ThunkInput.txt` and re-generated.